test/analysis/SmartDetectorsAnalysisTests/AlertPresentationTests.cs

```diff
@@ -356,6 +366,12 @@ public PresentationTestAlert(string title = "AlertTitle", ResourceIdentifier res
                     IgnoreDataBefore = new DateTime(1972, 6, 6),
                 }
             };
+
+            [TextProperty("DateTimeDisplayName", Order = 9, FormatString = "The time is: {0:yyyy-MM-dd HH}")]
+            public DateTime DateTimeValue => new DateTime(2019, 3, 7, 11, 23, 47);
+
+            [LongTextProperty("DoubleDisplayName", Order = 10, FormatString = "The area of the unit circle is {0:F0} in Indiana and {0:F6} anywhere else")]
```

Comment:
> "The area of the unit circle is {0:F0} in Indiana and {0:F6} anywhere else")] [](start = 77, length = 78)

this might be offensive to Indianasians...